geomstats/geometry/discrete_surfaces.py

```diff
@@ -1154,6 +1155,43 @@ def geodesic_ivp(self, tangent_vec, base_point):
         )
 
 
+class L2SurfacesMetric(NFoldMetric):
+    """L2 metric on the space of discrete surfaces."""
```

Comment:
NIT: missing description of base_manifold and n_copies attributes.

Reply:
Improved docstrings as a note (as these are not attributes of the metric).